web/src/app/administration/components/classification-list/classification-list.component.ts

```diff
@@ -118,4 +109,8 @@ export class ClassificationListComponent implements OnInit, OnDestroy {
     this.destroy$.next();
     this.destroy$.complete();
   }
+
+  setRequestInProgress(requestInProgress: boolean) {
```

Comment:
Is this mehtod really needed? Would a simple requestInProgress=$event in the html not suffice?

Reply:
true that.. done